include/ftl/task_scheduler.h

```diff
@@ -62,7 +66,7 @@ class TaskScheduler {
 	~TaskScheduler();
 
 private:
-	enum : size_t { FTL_INVALID_INDEX = UINT_MAX };
```

Comment:
Then this would be kInvalidIndex

Reply:
Let's just standardize on removing anonymous enums and just use static const.

Reply:
Caught in a later pass.